GliaWidgets/Window/GliaViewController.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit
+
+public enum GliaViewControllerEvent {
+    case minimized
+    case maximized
+}
+
+public protocol GliaViewControllerDelegate: AnyObject {
+    func event(_ event: GliaViewControllerEvent)
+}
+
+class GliaViewController: UIViewController {
+    var bubbleKind: BubbleKind = .userImage(url: nil) {
+        didSet { bubbleWindow?.bubbleKind = bubbleKind }
+    }
+
+    private enum State {
+        case maximized
+        case minimized
+    }
+
+    private var state: State = .maximized
```

Comment:
why do we need to retain the state? It's not being passed anywhere

Reply:
Most possibly also a leftover which I forgot to clean up

Reply:
Interestingly enough, there doesn't seem to be a use for that field in the earlier versions of the code either. Idk what's up with that